.github/workflows/triage.yml

```diff
@@ -3,136 +3,148 @@ name: Triage tasks
 on: pull_request_target
 
 concurrency:
-  group: "${{ github.ref }}"
+    group: "${{ github.ref }}"
 
 jobs:
-  triage:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check commit format
-        uses: Homebrew/actions/check-commit-format@master
```

Comment:
Too much indentation in this file :/

Reply:
@dawidd6 That's what the VS Code YAML formatter does by default.

Reply:
(it's consistent with all the other YAML files here)

Reply:
Whoops, no it's not, it's using 4 spaces rather than 2 for some reason? Pushing a fix shortly 😅

Reply:
Any chance we can get `brew style` to check style in `.yaml` files? There's a ton of inconsistency and it seems every few months there's a new PR that says "also reformat yaml file". It would be good if we can standardize that (and potentially even automatically fix)

Reply:
@Rylan12 Yeh, that'd be great! Would also accept similar on Markdown files (or anything else we use seriously).

Reply:
Mhm. VSCode acts weird with this particular file. Perhaps the embedded JSON has something to do with it.

I thought about factoring out this JSON blob into separate file, perhaps in `.github`. Whatcha say?

Reply:
@dawidd6 Sorry, missed this. Fine with me!